staging/src/k8s.io/apiserver/pkg/server/filters/waitgroup.go

```diff
@@ -42,7 +42,7 @@ func WithWaitGroup(handler http.Handler, requestContextMapper apirequest.Request
 
 		if !longRunning(req, requestInfo) {
 			if err := wg.Add(1); err != nil {
-				http.Error(w, "Apisever is shutting down.", http.StatusInternalServerError)
+				http.Error(w, "APIServer is shutting down.", http.StatusInternalServerError)
```

Comment:
du we have a unified spelling for this?

Reply:
I can find `apiserver` and `APIServer`, as you say the former looks better.